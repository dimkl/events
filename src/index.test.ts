import * as modules from "./index";

describe("index", () => {
  test("verified exports of module", () => {
    expect(Object.keys(modules).sort()).toMatchSnapshot();
  });
});
