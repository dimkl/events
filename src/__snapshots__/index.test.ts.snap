// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index verified exports of module 1`] = `
[
  "CustomEvent",
  "EventBus",
  "dispatch",
  "eventBus",
  "on",
]
`;
